src/radical/pilot/agent/lm/base.py

```diff
@@ -51,9 +53,9 @@ class LaunchMethod(object):
     #        cud.pre_exec, and then should also export those.  That would make
     #        our launch script ore complicated though...
     EXPORT_ENV_VARIABLES = [
-        'LD_LIBRARY_PATH',
-        'PATH',
-        'PYTHONPATH',
+      # 'LD_LIBRARY_PATH',
```

Comment:
Might this be related to the issues we discussed about missing envs on the compute nodes?

Reply:
Yes - but that wasn't on this branch... :-/  I'll revert this anyway.